e2e/test/scenarios/embedding/reproductions/37914-embed-preview-parameter-values-not-working.cy.spec.js

```diff
@@ -0,0 +1,173 @@
+import {
+  restore,
+  popover,
+  visitDashboard,
+  addOrUpdateDashboardCard,
+  dashboardHeader,
+  modal,
+} from "e2e/support/helpers";
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+
+const { PRODUCTS, PRODUCTS_ID } = SAMPLE_DATABASE;
+
+const questionDetails = {
+  name: "Products",
+  query: { "source-table": PRODUCTS_ID },
+};
+
+const filter3 = {
```

Comment:
I declare `filter3` first because I want to use `filter2`'s id in `filter`, and I don't want to hardcode that, so I need to declare `filter2` before `filter1`, and I figure I should put `filter3` first for consistency.